### PR TITLE
ui: add a new column to in volume list to indicate if the volume is bound

### LINKED ISSUE #1620: Add a new column indicating if the volume is bound
**Component**: ui, storage

**Why this is needed**:
As a user, I want to check if the volume is bound.

**What should be done**:
Add a new column `bound` in the volume list table. It should display either `Yes` or `No`.

**Implementation proposal** (strongly recommended):
Check with the status of PV. If it is `bound` then display `Yes` otherwise display `No`.

**Test plan**:
TBD

### PR DESCRIPTION
**Pull request by ChengYanJin**
**Component**: ui, storage

**Context**: 
As a user, I would like to know if the volume is bound.

**Summary**:
Add a new column in volume list, `bound`. 
When the persistent volume is bound, it displays `Yes/Oui`. Otherwise, it displays `No/Non`.

**Acceptance criteria**: 
The ui should look like the following screenshot.
image

Closes: #1620

### CODE REVIEW

File: ui/src/containers/NodeInformation.js [hunk 1]
```diff
@@ -10,29 +10,29 @@ import { fetchPodsAction } from '../ducks/app/pods';
 import { refreshNodesAction, stopRefreshNodesAction } from '../ducks/app/nodes';
 import {
   fetchVolumesAction,
-  fetchPersistentVolumeAction
+  fetchPersistentVolumeAction,
```
Comment (slaperche-scality) [inline ui/src/containers/NodeInformation.js:13]: I think all those trailing commas were added by the new linter settings Patrick added recently.

Ideally they would be in a dedicated commit:
- one commit for the linting/auto formatting
- one commit for your feature

Would make the review easier (you can blind approve the linter commit, and read the feature commit carefully without the noise).

File: ui/src/containers/NodeInformation.js [hunk 2]
```diff
@@ -169,21 +169,24 @@ const NodeInformation = props => {
...
-      pV => pV.metadata.name === volume.metadata.name
+      pV => pV.metadata.name === volume.metadata.name,
     );
-
     return {
       name: volume.metadata.name,
       status:
         (volume && volume.status && volume.status.phase) || STATUS_UNKNOWN,
+      bound:
+        volumePV && volumePV.status && volumePV.status.phase === STATUS_BOUND
```
Comment (slaperche-scality) [inline ui/src/containers/NodeInformation.js]: don't know if this is possible, but can it be simplified as: `volumePV?.status?.phase === STATUS_BOUND`?

File: ui/src/containers/NodeInformation.js (hunk 1 above)
Comment (slaperche-scality) [reply ui/src/containers/NodeInformation.js:13]: I can provide Git guidance to do so if needed :slightly_smiling_face:

**Comment by slaperche-scality** [comment]
Tested: it works as advertised :slightly_smiling_face:

**Review by MonPote — APPROVED** [review]

**Comment by ChengYanJin** [comment]
@bert-e approve

**Review by slaperche-scality — APPROVED** [review]
